Packs/VMwareWorkspaceONEUEM/Integrations/VMwareWorkspaceONEUEM/VMwareWorkspaceONEUEM.py

```diff
@@ -0,0 +1,644 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Tuple, List, Any, Optional
+
+# Disable insecure warnings
+# requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+READABLE_DATE_FORMAT = '%B %d, %Y at %I:%M:%S %p'
+API_VERSION = 2
+LOGGING_INTEGRATION_NAME = "[VMware Workspace ONE UEM (AirWatch MDM)]"
+URL_SUFFIX = {
```

Comment:
Let's use the suffix_url in the requests itself. No need for global here.

Reply:
@Shellyber, This is something we're following as a best practice to create a common URL_SUFFIX. It benefits us to make changes to the endpoint URL if used in multiple methods.
Let us know if you want us to make this change anyway.

Reply:
@crestdatasystems yes, let's change that please. 
It is more easier to read without this enums. 
Also let's change it to `OUTPUT_PREFIX`

Reply:
@Shellyber okay, will make this change